### PR TITLE
Expect SHA-256 instead of MD5 fingerprint flag value

### PR DESCRIPTION
**Pull request by omorsi**
This PR solves issue #156.

Copying the issue description here to save readers' time.
------
This is to complete pull request #144 which should have removed all weak hash functions usages.

https://github.com/micromdm/scep/blob/c89253406d2e73be69933382e1c0aed9069283be/cmd/scepclient/scepclient.go#L271

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ cat - > /tmp/scep.csr
 $ ./scepclient-linux-amd64 -help
 Usage of ./scepclient-linux-amd64:
   -ca-fingerprint string
-    	md5 fingerprint of CA certificate for NDES server.
+    	(Note: Changed from MD5) SHA-256 digest of CA certificate for NDES server.
```
Comment (jessepeterson) [inline README.md:122]: nit: see note below

File: cmd/scepclient/scepclient.go [hunk 2]
```diff
@@ -268,7 +269,7 @@ func main() {
 
 		// in case of multiple certificate authorities, we need to figure out who the recipient of the encrypted
 		// data is.
-		flCAFingerprint = flag.String("ca-fingerprint", "", "md5 fingerprint of CA certificate for NDES server.")
+		flCAFingerprint = flag.String("ca-fingerprint", "", "(Note: Changed from MD5) SHA-256 digest of CA certificate for NDES server.")
```
Comment (jessepeterson) [inline cmd/scepclient/scepclient.go:272]: nit: place note behind new description

**Review by jessepeterson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jessepeterson) [reply README.md:122]: Also, thanks for remember to update the docs! :)

Comment (omorsi) [reply README.md]: Done. thanks :)

File: cmd/scepclient/scepclient.go (hunk 2 above)
Comment (omorsi) [reply cmd/scepclient/scepclient.go]: Done.

**Pull request merged**